packages/dev/core/src/Meshes/mesh.ts

```diff
@@ -661,7 +662,7 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
 
             // Physics clone
             if (scene.getPhysicsEngine) {
-                const physicsEngine = scene.getPhysicsEngine();
+                const physicsEngine = scene.getPhysicsEngine() as PhysicsEngineV1;

```

Comment:
Shouldn't we check if the engine is v1 before casting?

Reply:
Fixed!